.github/workflows/linux.yml

```
@@ -0,0 +1,186 @@
+name: Linux
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  main:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - cubit: 17_1_0
+            os: ubuntu-18.04
+            URL: LINUX_CUBIT_17_1_0
+            URL_SDK: LINUX_CUBIT_SDK_17_1_0
+            
+          - cubit: 2020_2
+            os: ubuntu-20.04
+            URL: LINUX_CUBIT_2020_2
+            URL_SDK: LINUX_CUBIT_SDK_2020_2
+
+    name: 'Trelis/Cubit Svalinn Plugin ${{ matrix.cubit }} Build for Linux'
+
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Environment Variables
+        run: |
+          echo "CURRENT=$(pwd)" >> $GITHUB_ENV
+          echo "SCRIPTPATH=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          echo "PLUGIN_ABS_PATH=$GITHUB_WORKSPACE/.." >> $GITHUB_ENV
+          if [ "${{ matrix.cubit }}" = "17_1_0" ]; then
+            echo "CUBIT_PATH=/opt/Trelis-17.1" >> $GITHUB_ENV
+            echo "CMAKE_ADDITIONAL_FLAG=-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0" >> $GITHUB_ENV
```

I'm assuming this variable will be used to capture any additional CMake flags we pass to these commands in the future, so I'll suggest we make it plural

```suggestion
            echo "CMAKE_ADDITIONAL_FLAGS=-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0" >> $GITHUB_ENV
```